examples/HowTo.md

```diff
@@ -0,0 +1,147 @@
+# `log-warper` how-to
+
+If you want to see the working example first, please, check out this [very simple example](https://github.com/serokell/log-warper/blob/master/examples/Playground.hs).
+
+Here we will try to explain step by step how to integrate `log-warper` into your project.
+
+All code below can be compiled and run with these commands
+
+```bash
+stack build log-warper
+stack exec how-to
+```
+and you can see how it's working both with and without logging.
+
+Since this is a literate haskell file, we need to specify all our language extensions and imports up front.
+
+``` haskell
+{-# LANGUAGE OverloadedStrings #-}
+
+module Main (main) where
+
+
+import Control.Monad.IO.Class (MonadIO, liftIO)
+import Data.Monoid ((<>))
+import Data.Text (Text)
+import System.Wlog (WithLoggerIO, buildAndSetupYamlLogging, logError,
+                    logInfo,productionB, usingLoggerName)
+
+import qualified Data.Text as T
+import qualified Data.Text.IO as TIO
+
+```
+
+## Adding a new logger
+
+First of all, you may ask what are we doing? What the hell is logger?
+
+**Logger** is the [output handler](https://hackage.haskell.org/package/log-warper-1.7.1/docs/System-Wlog-LogHandler.html#t:LogHandler) labeled by the unique [logger name](https://hackage.haskell.org/package/log-warper-1.7.1/docs/System-Wlog-LoggerName.html#t:LoggerName).
+
+To add a new logger for your needs you'll have to consider several things.
+Firstly, you should consider choosing some reasonable name for the logger you creating according to the actions that it would cover with logging. From now on let's assume that you need to have `new-logger` in your project.
+
+### Configuration file
+
+To configure your logger you will need configuration `.yaml` file.
+In our example we will use very simple one with the minimal settings needed for the file to work:
+
+```yaml
+loggerTree:
+    severity: Warning+    # severities for «root» logger
+    new-logger:           # logger named «new-logger»
+        severity: Debug+  # severities for logger «new-logger»
+```
+
+For full information about all configuration file's features and options please see [this section](#link-to-readme-config).
```

Comment:
I think that you also should add that user don't need to worry about `root` logger for now. This part is explained in section devoted to hierarchical logging.